### PR TITLE
Refs #12970 - Package sass-rails 5 for Patternfly

### PR DESCRIPTION
**Pull request by dLobatog**

### CODE REVIEW

File: foreman/foreman.spec [hunk 1]
```diff
@@ -187,8 +187,8 @@ BuildRequires: %{?scl_prefix_ror}rubygem(sprockets-rails) < 3
 # assets
 BuildRequires: %{?scl_prefix}rubygem(ace-rails-ap) >= 4.0.0
 BuildRequires: %{?scl_prefix}rubygem(ace-rails-ap) < 4.1.0
-BuildRequires: %{?scl_prefix_ror}rubygem(sass-rails) >= 4.0
-BuildRequires: %{?scl_prefix_ror}rubygem(sass-rails) < 5.0
+BuildRequires: %{?scl_prefix_ror}rubygem(sass-rails) >= 5
```
Comment (domcleal) [inline foreman/foreman.spec]: If you're moving it into our collection, `scl_prefix` and not _ror.

File: rubygem-sass-rails/rubygem-sass-rails.spec [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+Requires: %{?scl_prefix_ruby}ruby
+Requires: %{?scl_prefix_ror}rubygem(sass) >= 3.1
+Requires: %{?scl_prefix_ror}rubygem(sass) < 4
+Requires: %{?scl_prefix_ror}rubygem(railties) >= 4
+Requires: %{?scl_prefix_ror}rubygem(railties) < 5
+Requires: %{?scl_prefix}rubygem(sprockets) >= 2.8
+Requires: %{?scl_prefix}rubygem(sprockets) < 4
+Requires: %{?scl_prefix}rubygem(sprockets-rails) >= 4
+Requires: %{?scl_prefix}rubygem(sprockets-rails) < 2
+Requires: %{?scl_prefix}rubygem(tilt) >= 1.1
```
Comment (domcleal) [inline rubygem-sass-rails/rubygem-sass-rails.spec]: tilt's in ror?

**Comment by domcleal** [comment]
This spec file should be imported from an existing SCL or Fedora, not rewritten if possible, as we lose the tests, history etc.  Here's an SCLed copy of 5.0.4 if it helps - https://github.com/domcleal/sclo-ror42/blob/master/rubygem-sass-rails/rubygem-sass-rails.spec

**Comment by dLobatog** [comment]
@domcleal I based the spec off the link you posted, so now it also has a check section. The only thing I'm not sure of now is whether this package should be 5.0.4-2, as 5.0.4-1 is already listed on the changelog.

**Comment by domcleal** [comment]
Yeah, -2's a good idea as the contents have changed.

**Comment by domcleal** [comment]
Please can you rebase this?  I think the build deps are in now.

**Comment by dLobatog** [comment]
@domcleal Rebased, conflicts were minimal on comps and tito.props

**Comment by domcleal** [comment]
Thanks @dLobatog, building now.

**Pull request merged**